### PR TITLE
[DPEDE-3079] Add visual tests to copy text component

### PR DESCRIPTION
**Pull request by xft-dev**
https://lumen.atlassian.net/browse/DPEDE-3079

Test it with https://github.com/CenturyLink/chi-custom-elements/pull/58

Problem: The component had display block and break a line. See image:
image

### CODE REVIEW

File: test/chi/components/copy-text.pug [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
```
Comment (wojciechwlodarczyklumen) [inline test/chi/components/copy-text.pug:1]: Shoudn't that file contain HTML representation of this component ?

File: test/chi/components/copy-text.pug [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+title: Copy Text
+---
+script(type='module' src='../../js/ce/ux-chi-ce/ux-chi-ce.esm.js')
```
Comment (ASanchLumen) [inline test/chi/components/copy-text.pug:4]:
```suggestion
```

File: test/chi/components/copy-text.pug (hunk 1 above)
Comment (ASanchLumen) [reply test/chi/components/copy-text.pug:1]: Yeah!! good catch

**Comment by ASanchLumen** [comment]
Needs to be updated with master to take the visual tests for date-picker

**Review by josecarlosgonzalezv — CHANGES_REQUESTED** [review]
This needs to be rebased with the latests changes on `master`. 
The fix is currently available.

**Comment by ASanchLumen** [comment]
Reopened in new PR https://github.com/CenturyLink/Chi/pull/1459

**Comment by ASanchLumen** [comment]
Reopened in new PR https://github.com/CenturyLink/Chi/pull/1459

**Pull request closed without merging**